lib/tasks/email_giftcards.rake

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+namespace :email_giftcards do
```

Comment:
Optional, but if you want you can put this in the `emailer.rake` namespace where the other task was. That way the tasks can be called something like `rake emailer:vouchers_to_merchants` or `emailer:vouchers_to_customers` or maybe even just rename it `email` to make it even more succinct. I trust you to choose what's best!